website/docs/guides/container_service_extension_4_0.html.markdown

```diff
@@ -0,0 +1,411 @@
+  configuration (called "VCDKEConfig") that will be instantiated in next step with a [RDE][rde].
+- The required `capvcdCluster` [RDE Type][rde_type]. Its version is specified by the `capvcd_rde_version` variable, that **must be "1.1.0" for CSE v4.0**.
+  This resource specifies the schema of the TKGm clusters.
+- The **CSE Admin [Role][role]**, that specifies the required rights for the CSE Administrator to manage provider-sided elements of VCD.
+- The **CSE Administrator [User][user]** that will administrate the CSE Server and other aspects of VCD that are directly related to CSE.
+  Feel free to add more attributes like `description` or `full_name` if needed.
+
+Once reviewed and applied with `terraform apply`, one **must login with the created CSE Administrator user to
+generate an API token** that will be used in the next step. In UI, the API tokens can be generated in the CSE Administrator
+user preferences in the top right, then go to the API tokens section, add a new one.
+Or you can visit `/provider/administration/settings/user-preferences` from your VCD url as CSE Administrator.
+
+### Step 2: Install CSE
+
+-> This step of the installation refers to the Terraform configuration present [here][step2].
+
+~> Be sure that previous step is successfully completed and the API token for the CSE Administrator user was created.
+
+This step will create all the remaining elements to install CSE v4.0 in VCD. You can read subsequent sections
+to have a better understanding of the building blocks that are described in the [proposed Terraform configuration][step2].
+
+In this [configuration][step2] you can also find a file named `terraform.tfvars.example`, you need to rename it to `terraform.tfvars`
+and change the values present there to the correct ones. You can also modify the proposed resources so they fit better to your needs.
+
+#### Organizations
+
+The [proposed configuration][step2] will create two new [Organizations][org], as specified in the [CSE documentation][cse_docs]:
+
+- A Solutions [Organization][org]: This [Organization][org] will host all provider-scoped items, such as the CSE Server.
+  It should only be accessible to the CSE Administrator and Providers.
+- A Tenant [Organization][org]: This [Organization][org] will host the TKGm clusters for the users of this tenant to consume them.
+
+If you already have these two [Organizations][org] created and you want to use them instead, you can leverage customising the [proposed configuration][step2]
+to use the Organization [data source][org_d] to fetch them.
+
+#### VM Sizing Policies
+
+The [proposed configuration][step2] will create four VM Sizing Policies:
+
+- `TKG extra_large`: 8 CPUs, 32GB RAM.
+- `TKG large`: 4 CPUs, 16GB RAM.
+- `TKG medium`: 2 CPUs, 8GB RAM.
+- `TKG small`: 2 CPU, 4GB RAM.
+
+These VM Sizing Policies should be applied as they are. Nothing should be changed here. They will be assigned to the Tenant
```

Comment:
> Nothing should be changed here.

First question that pops up for me after reading this is "why?" :) If there is a short way explaining why it shouldn't be changed, please add a sentence.